integration-tests/src/test/scala/za/co/absa/spline/ExcelSpec.scala

```diff
@@ -25,12 +25,15 @@ import za.co.absa.commons.io.{TempDirectory, TempFile}
 import za.co.absa.spline.test.fixture.SparkFixture
 import za.co.absa.spline.test.fixture.spline.SplineFixture
 
+import java.io.File
+
 class ExcelSpec extends AsyncFlatSpec
   with Matchers
   with SparkFixture
   with SplineFixture {
 
   private val filePath = TempFile("file1", ".xlsx", false).deleteOnExit().path.toAbsolutePath.toString
+  private val fileUri = new File(filePath).toURI.toString
```

Comment:
minor:
unnecessary `toString` call. The name `fileUri` suggests it is of the `URI` type, while in fact is `String`. If would be OK in other situations where it's passed to the expressions where a string is expected. But in this particular context (and the other similar ones in this PR) the given variable is only used in string interpolation, where `Any` is expected and `toString` is called implicitly. So it would be more natural for `fileUri` variable to be of the `URI` type.

Reply:
maybe `toURI` method should be added to `TempFile` and `TempDirectory` so you can call 
```scala
private val fileUri = TempFile("file1", ".xlsx", false).deleteOnExit().toURI
```